src/plugins/discover/public/application/main/components/layout/types.ts

```diff
@@ -6,6 +6,7 @@
  * Side Public License, v 1.
  */
 
+import { ElasticSearchHit } from 'src/plugins/discover/public/services/doc_views/doc_views_types';
```

Comment:
unrelated to this PR , but since we're using ElasticSearchHit at several place in Discover we could create e.g. create a `types.ts` here: https://github.com/elastic/kibana/tree/main/src/plugins/discover/public WDYT?

Reply:
anyway, this wouldn't be a mandatory change, but I think the link to the types should be relative like the other imports here.

Reply:
Changed.